frontend/www/js/omegaup/group/edit.ts

```diff
@@ -0,0 +1,281 @@
+import group_Edit, { AvailableTabs } from '../components/group/Edit.vue';
+import group_Members from '../components/group/Members.vue';
+import group_Scoreboards from '../components/group/Scoreboards.vue';
+import group_Identities from '../components/group/Identities.vue';
+import { OmegaUp } from '../omegaup';
+import { types } from '../api_types';
+import * as api from '../api';
+import * as ui from '../ui';
+import T from '../lang';
+import Vue from 'vue';
+import * as CSV from '@/third_party/js/csv.js/csv.js';
+
+OmegaUp.on('ready', () => {
+  const payload = types.payloadParsers.GroupEditPayload();
+  const groupEdit = new Vue({
+    el: '#main-container',
+    components: {
+      'omegaup-group-edit': group_Edit,
+    },
+    data: () => ({
+      tab: window.location.hash
+        ? window.location.hash.substr(1)
+        : AvailableTabs.Members,
+      identities: payload.identities.filter(
+        (identity) => identity.username.split(':').length === 1,
+      ),
+      identitiesCsv: payload.identities.filter(
+        (identity) => identity.username.split(':').length !== 1,
+      ),
+      scoreboards: payload.scoreboards,
+    }),
+    methods: {
+      refreshGroupScoreboards: (): void => {
+        api.Group.details({ group_alias: payload.groupAlias })
+          .then((group) => {
+            groupEdit.scoreboards = group.scoreboards;
+          })
+          .catch(ui.apiError);
+      },
+      refreshMemberList: (): void => {
+        api.Group.members({ group_alias: payload.groupAlias })
+          .then((data) => {
+            groupEdit.identities = data.identities.filter(
+              (identity) => identity.username.split(':').length === 1,
```

Comment:
```suggestion
              (identity) => identity.username.includes(':'),
```

cortesía de [`String.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes). y ahí en esa misma página viene el polyfill.

igual abajo.

Reply:
Listo. También agregué el polyfill